Source/SwiftLintFrameworkTests/YamlParserTests.swift

```diff
@@ -0,0 +1,30 @@
+//
```

Comment:
My mistake again. I had completely missed `YamlParserError` when reviewing. This looks good.